### DISCUSSION TITLE
Lowdefy icon problem

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#903] [Questions and Help] [answered]
Hello, I have a question. Can the main icon of the lowdefy menu be changed during the development process, or can it be changed only after the Department has signed it?

**Comment by Gervwyk** [comment, marked as answer]
HI @xiaoxiji1234 

I'm not sure I understand your question. Could you please provide more detail? perhaps with an example?

**Reply by xiaoxiji1234** [reply]
Hello, can lowdefy's logo be changed in the development environment?
![image](https://user-images.githubusercontent.com/72787010/137834636-5890ddc6-99ed-4705-af21-709d0cd94b8c.png)

**Reply by SamTolmay** [reply]
You can change the logo by creating a folder called `public` in your project directory, and adding image files with the correct names to this folder. The names of the files are documented here:
https://docs.lowdefy.com/hosting-files

**Reply by xiaoxiji1234** [reply]
Ok, I got it, Thank you!
